File: Task-4/ReverseNumber.java
class ReverseNumber
{    
  static  int reverse(int number) {
    int reversed = 0;
    while(number != 0) {  
      int remainder = number % 10;  
      reversed = reversed * 10 + remainder;  
      number = number/10;  
      }
    return reversed;
 
  }
  public static void main(String args[])
  {
    System.out.println(reverse(123)); // Output = 321
  }
}

File: Task-4/ReverseNumber.py
def reverse_number(num):
    reverseNum=0
    while num>0:
        quotient=num%10
        remainder=num//10
        reverseNum=(reverseNum*10)+quotient
        num=remainder
    return(reverseNum)

print("Reverse of number 123 is : ",reverse_number(123))
